01week/bio/index.html

```diff
@@ -2,8 +2,11 @@
 <html>
```

Comment:
Remember to label all of your files <!DOCTYPE html> when opening a new file. When in atom if you just type html and press enter it automatically fills in all of your opening tags :)